docs/Readme.md

```diff
@@ -9,7 +9,6 @@ The `src` folder contains the following documentation sources:
 
 - `src/api-v1`: The DSP-API V1 (JSON) Request and Response Format documentation. Source can be found in `salsah1/src/typescript_interfaces`
 - `src/api-v2`: The DSP-API V2 (JSON-LD) Request and Response Format documentation.
-- `src/api-admin`: The DSP-API Admin (JSON) Request and Response format Swagger-based documentation.
```

Comment:
This whole structure is not up-to-date. Could you go over the whole `Readme.md` file if you touch it anyway?

Reply:
Although it's out of the scope of this PR I cleaned it up. There should be another task defined to clean the whole API documentation though.

Reply:
And I removed too much :)
Do we actually need this "test"?
```
  test-docs-build:
    name: Test docs
    runs-on: ubuntu-latest
    steps:
      - name: Checkout source
        uses: actions/checkout@v3
      - name: Install requirements (pip, npm, apt)
        run: |
          python -m pip install --upgrade pip
          pip install -r docs/requirements.txt
          npm install --global typedoc
          sudo apt-get install graphviz
      - name: Build docs
        run: make docs-build
```

Reply:
Yes, we need it. It's to make sure that the docs compile.